x/mongo/driver/topology/server.go

```diff
@@ -546,7 +546,7 @@ func (s *Server) createConnection() (*connection, error) {
 		WithReadTimeout(func(time.Duration) time.Duration { return s.cfg.heartbeatTimeout }),
 		WithWriteTimeout(func(time.Duration) time.Duration { return s.cfg.heartbeatTimeout }),
 	}
-	opts = append(opts, s.cfg.connectionOpts...)
+	opts = append(s.cfg.connectionOpts, opts...)
```

Comment:
I know the code was already this way, but can we move `WithHandshaker` and `WithMonitor` into the `opts` slice declaration above and have this `append` line be directly after?

Reply:
done